### PR TITLE
fix: crash under csp

### PR DESCRIPTION
**Pull request by Jack-Works**
close https://github.com/endojs/endo/issues/1281

### CODE REVIEW

**Review by kriskowal — COMMENTED** [review]
Freeze would suffice here

File: packages/ses/src/tame-function-constructors.js [hunk 1]
```diff
@@ -49,7 +49,7 @@ export default function tameFunctionConstructors() {
     FERAL_FUNCTION.prototype.constructor('return 1');
   } catch (ignore) {
     // Throws, no need to patch.
-    return harden({});
+    return {};
```
Comment (kriskowal) [inline packages/ses/src/tame-function-constructors.js:52]:
```suggestion
    return Object.freeze({});
```

Comment (kriskowal) [reply packages/ses/src/tame-function-constructors.js:52]: Will also need to import freeze from commons.js

**Comment by mhofman** [comment]
Not in scope for this PR, but wondering what kind of test we could have to make sure we don't break no-eval environments. The setup would likely look like replacing the evaluators with functions that throw before importing ses. The question is how to scope a subset of the tests with this environment.

**Comment by Jack-Works** [comment]
since Compartment is not available under no-eval environment, test lockdown is enough.

**Comment by kriskowal** [comment]
> Not in scope for this PR, but wondering what kind of test we could have to make sure we don't break no-eval environments. The setup would likely look like replacing the evaluators with functions that throw before importing ses. The question is how to scope a subset of the tests with this environment.

As it happens, you wrote one already. https://github.com/endojs/endo/blob/master/packages/ses/test/test-no-eval.js

It’s marked as `failing`. I’d like this PR to remove the `failing` bit from that test. I looked into it and this change isn’t sufficient. Thankfully, @mhofman left notes: https://github.com/endojs/endo/issues/903

```
  no-eval › lockdown must not throw when eval is forbidden

  Error thrown in test:

  TypeError {
    message: 'no unsafe-eval, as if by content-security-policy',
  }

  › globalThis.eval (file://test/no-eval.js:6:9)
  › repairFunction (file://src/tame-function-constructors.js:71:35)
  › tameFunctionConstructors (file://src/tame-function-constructors.js:118:3)
  › repairIntrinsics (file://src/lockdown-shim.js:276:17)
  › lockdown (file://src/lockdown-shim.js:410:28)
  › file://test/test-no-eval.js:9:30
```

**Review by kriskowal — APPROVED** [review]

**Comment by kriskowal** [comment]
I’m landing this as it’s incremental progress. Further work will be necessary to avoid eval with a lockdown flag without compromising lockdown invariants.

**Pull request merged**